### PR TITLE
test: Add test for _mm_store_ss

### PR DESCRIPTION
**Pull request by HowJMay**

### CODE REVIEW

File: tests/impl.cpp [hunk 1]
```diff
@@ -1991,7 +1991,13 @@ result_t test_mm_store_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_store_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    float *p = (float *) impl.mTestFloatPointer1;
+    float x = impl.mTestFloats[i + 4];
```
Comment (marktwtn) [inline tests/impl.cpp:1995]:
```
float x = impl.mTestFloats[i];
float p[4];
```

I think we should not store the value into input data.

**Review by marktwtn — CHANGES_REQUESTED** [review]

File: tests/impl.cpp (hunk 1 above)
Comment (HowJMay) [reply tests/impl.cpp:1995]: I refered to `test_mm_store_ps()`
`impl.mTestFloats` is taking a random float as testing data

Comment (marktwtn) [reply tests/impl.cpp:1995]: You mean referred from `test_mm_store_ps()`?
We still have random float in this modification.

Comment (HowJMay) [reply tests/impl.cpp:1995]: Is there any suggested way to get a random testing number

Comment (marktwtn) [reply tests/impl.cpp:1995]: We already have it.
The `impl.mTestFloats` array is generated with random float values.

Comment (HowJMay) [reply tests/impl.cpp:1995]: so I can just simply use `impl.mTestFloats[0]`?

Comment (marktwtn) [reply tests/impl.cpp:1995]: Use `impl.mTestFloats[i]`.

Comment (HowJMay) [reply tests/impl.cpp:1995]: got it

File: tests/impl.cpp [hunk 2]
```diff
@@ -1991,7 +1991,13 @@ result_t test_mm_store_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_store_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    float *p = (float *) impl.mTestFloatPointer1;
```
Comment (marktwtn) [inline tests/impl.cpp:1994]: Rewrite it as `float p[4];`.

**Review by marktwtn — CHANGES_REQUESTED** [review]

**Review by marktwtn — APPROVED** [review]

**Pull request merged**